File: fintechplatform/src/main/java/com/dwaplatform/android/money/MoneyHelperModule.java
package com.dwaplatform.android.money;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;

/**
 * Created by ingrid on 08/12/17.
 */

@Module
public class MoneyHelperModule {
    @Provides
    @Singleton
    public MoneyHelper providesMoneyHelper() {
        return new MoneyHelper();
    }
}

File: dwaplatform/src/main/java/com/dwaplatform/android/money/MoneyHelperModule.kt
package com.dwaplatform.android.money


import javax.inject.Singleton

import dagger.Module
import dagger.Provides

/**
 * Created by tcappellari on 08/12/2017.
 */
@Module
class MoneyHelperModule {
    @Provides
    @Singleton
    internal fun provideMoneyHelper(): MoneyHelper {
        return MoneyHelper()
    }
}
